book/src/project-policies.md

```diff
@@ -22,6 +22,10 @@ A litmus test for not breaking compatibility is to replace the image of a
 component in an existing deployment without changing that deployment in any
 other way.
 
+To minimize the number of branches we need to support, we do not have a general
+policy for releasing new minor versions on older majors. We will make exceptions
+for work related to meeting production readiness requirements.
+
```

Comment:
We should add a clause that this older major can only be one version lower than the current major and also the time between the two major releases is less than 3-4 months, etc.

Reply:
Done. Does the wording above make sense?

Reply:
i'm not sure how to read this. Are we trying to say that if "X.0.0" and "X+1.0.0" were released about a quarter apart, then we will consider some exception?

But if it was "X.2.0" and "X+1.0.0", then we won't consider any exception?

Reply:
Something like that.  We only look at the initial release times of the major releases for minor release exceptions.  Otherwise we could end up with too many releases.  For example, if X.0.0 was released a year before X+1.0.0, we should move on and not to keep releasing minor releases on X.

Reply:
Does the wording in this PR capture this in an understandable way? Or what if we worded it like so?

> Only prior major versions released within four months of the current major version will be eligible for this exception.

This wording does broaden it to more than one major release, but maybe it's more intuitive than the current wording.

Reply:
I think the wording as is captures it, but I wanted to confirm the intended scenario. Maybe give an example with numbers to make it more clear.

Reply:
Done

Reply:
Just checking if this captures everything. Do we also want to add a section for RBAC not requiring new majors?

Reply:
I think RBACs still require some more discussion, so let's consider that separately.